<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login - Event Check-in Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="lblLogo">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>51</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>üíª</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
font: 75 &quot;Nirmala UI&quot;;
font: 22pt &quot;Palatino Linotype&quot;;</string>
      </property>
      <property name="text">
       <string>Event Check-in Management</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblSubtitle">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #7f8c8d; font-size: 18px; margin-bottom: 24px;
font: 18pt &quot;Palatino Linotype&quot;;

color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>LOGIN TO CONTINUE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="cardWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#cardWidget {
    background-color: white;
    border-radius: 18px;
    border: 3px solid #dfe6e9;
}</string>
      </property>
      <layout class="QVBoxLayout" name="cardLayout">
       <property name="spacing">
        <number>14</number>
       </property>
       <property name="leftMargin">
        <number>28</number>
       </property>
       <property name="topMargin">
        <number>28</number>
       </property>
       <property name="rightMargin">
        <number>28</number>
       </property>
       <property name="bottomMargin">
        <number>28</number>
       </property>
       <item>
        <widget class="QLabel" name="lblUsername">
         <property name="styleSheet">
          <string notr="true">font-weight: bold; color: #2c3e50; font-size: 16px;
background-color: rgb(255, 255, 255);
color: rgb(85, 85, 255);</string>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUsername">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPassword">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold; color: #2c3e50; font-size: 16px;
background-color: rgb(255, 255, 255);
color: rgb(85, 85, 255);</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTogglePwd">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: #ecf0f1;
    border-radius: 6px;
    border: 1px solid #bdc3c7;
    font-size: 16px;
}
QPushButton:hover {
    background: #d5dbdb;
}</string>
           </property>
           <property name="text">
            <string>üëÅ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxRemember">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Remember password</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnForgotPassword">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    color: #3498db;
    font-size: 12px;
    border: none;
    background: transparent;
}
QPushButton:hover {
    color: #2980b9;
    text-decoration: underline;
}</string>
           </property>
           <property name="text">
            <string>Forgot password</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">color: #ecf0f1;</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogin">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border-radius: 8px;
    border: 12px;
	border-color: rgb(0, 0, 0);
}
QPushButton:hover {
    background-color: #27ae60;
}
QPushButton:pressed {
    background-color: #1e8449;
}</string>
         </property>
         <property name="text">
          <string>  üîë  Login</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: rgb(255, 0, 0);
    color: rgb(0, 0, 0);
    font-size: 16px;
    border-radius: 8px;
    border: none;
}

</string>
         </property>
         <property name="text">
          <string>‚ùå Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblVersion">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0); font-size: 10px; margin-top: 14px;</string>
      </property>
      <property name="text">
       <string>v1.0.0 ‚Äì Event Management System ¬© 2026</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
